docs/articles/nunit/getting-started/installation.md

```diff
@@ -1,43 +1,149 @@
+
+## I want to do it the old and hard way
+
+This is from the older documentation, and is still valid, but we recommend you use the above methods.
 
 1. Full NUnit install via NuGet.
 2. NUnitLite install via NuGet.
 3. Zip and/or MSI file download.
 4. Combined Approach
 
-## Using NuGet Packages
+### Using NuGet Packages and the NUnit Console
 
 In Visual Studio, from the Tools menu, select NuGet Package Manager | Manage NuGet packages for solution...
 Open the Browser tab, and Scroll or use search to locate the **NUnit** and **NUnit.Console** packages.
 
-## NUnit and NUnit.Console
+### NUnit and NUnit.Console
 
 Install both packages. The **NUnit** package should be referenced by each of your test assemblies, but not by any others.
 
 Locate nunit3-console in the **packages\NUnit.ConsoleRunner.3.X.X\tools** (or your configured package directory of choice) directory under your solution. This is the location from which you must run nunit3-console when if you would like to run NUnit3 from console.
 We recommend you only use this approach when running under the control of a script on your build server.
 
-## NUnit3TestAdapter
+See here for more details of using the [NUnit Console app](https://github.com/nunit/nunit-console#readme)
 
-If you want to run NUnit tests automated on a clean machine without any installations (e.g. TFS build agent) - and you're using Visual Studio 2012 or later, use this package.
+## NUnit3TestAdapter
 
-It's based on [the Visual Studio Test Adapter](xref:vstestadapterinstallation) and provides a compiled NUnit3 Visual Studio Test Adapter.
+If you want to run NUnit tests automated on a clean machine without any installations (e.g. Azure Devops or Github actions build agent) - and you're using Visual Studio 2012 or later, use this package.
 
-### Using NuGet NUnitLite Package
+See [the Visual Studio Test Adapter](xref:vstestadapterinstallation) for details.
 
-The NUnitLite approach provides a way to run NUnit tests without a full install of the NUnit runner and test engine assemblies. Only the framework and a small runner program are installed. Note that this is currently the only way to run tests under SilverLight 5.0 or the compact framework.
+Note that if you have used any of the top 4 approaches, you already have this package installed, and it will work on any CI build system.
 
-In Visual Studio, from the Tools menu, select NuGet Package Manager | Manage NuGet packages for solution...
+### Using NuGet NUnitLite Package
 
-Scroll or use search to locate the **NUnitLite Version 3** and install it. The package should be referenced by each of your test assemblies, but not by any others.
+The NUnitLite approach provides a way to run NUnit tests without a full install of the NUnit runner and test engine assemblies. Only the framework and a small runner program are installed. If you want to experiment with NUnit, debug something in NUnit, this can be a good way to go.
```

Comment:
```suggestion
The NUnitLite approach provides a way to run NUnit tests without a full install of the NUnit runner and test engine assemblies. Only the framework and a small runner program are installed. If you want to experiment with NUnit or debug something in NUnit, this can be a helpful choice.
```